CHANGELOG

```diff
@@ -1,3 +1,14 @@
+## [0.3.0] - 2023-01-11
+### New Features
+ - VM under test and Traffic generator are now using the isolateEmulatorThread option
+ - logs were streamlined
+ - trex output is checked for errors
+ - Upgraded go version to 1.20.12
+ - Checkup base image was upgraded to `registry.access.redhat.com/ubi9/ubi-minimal:9.3-1475`
+### Bug Fixes
```

Comment:
We have more bugs solved, such as the trex output in case invalid params inserted. Do you want to add them as well?

Reply:
On line 5 there is a reference to it.